packages/theming/apple-theme/src/fluentAppleColors.ios.ts

```diff
@@ -0,0 +1,321 @@
+  warningShade10: ColorValue;
+  warningShade20: ColorValue;
+  warningShade30: ColorValue;
+  warningTint10: ColorValue;
+  warningTint20: ColorValue;
+  warningTint30: ColorValue;
+  warningTint40: ColorValue;
+}
+
+/** A set of colors that define semantic colors in FluentUI Apple */
+export interface FluentAppleSemanticColors {
+  /**
+   * text color should not be lower than `gray500` in light mode to achieve 4.5:1 minimum contrast ratio in `.white` background
+   * text color should not be higher than `gray400` in dark mode to achieve 4.5:1 minimum contrast ratio in `.black` background
+   * when determining high contrast color, add 200 in light mode and substract 200 in dark mode from the default color.
+   * text color used for main level in the screen. eg. title in dialog, title in navigationbar with `surfacePrimary`, etc
+   */
+
+  /** text color used for main level in the screen. eg. title in dialog, title in navigationbar with `surfacePrimary`, etc */
+  textDominant: ColorValue; //= UIColor(light: gray900, lightHighContrast: .black, dark: .white)
+  /** text color used for titles */
+  textPrimary: ColorValue; //= UIColor(light: gray900, lightHighContrast: .black, dark: gray100, darkHighContrast: .white)
+  /** text color used for subtitles */
+  textSecondary: ColorValue; //= UIColor(light: gray500, lightHighContrast: gray700, dark: gray400, darkHighContrast: gray200)
+  /** text color used in disabled state */
+  textDisabled: ColorValue; //= UIColor(light: gray300, lightHighContrast: gray500, dark: gray600, darkHighContrast: gray400)
+  /** text appears on top of the surface that uses `Colors.primary` as background color */
+  textOnAccent: ColorValue; //= UIColor(light: .white, dark: .black)
+
+  /** icon used as call-to-actions without a label attached. They need to reach a minimum contrast ratio 4.5:1 to its background */
+  iconPrimary: ColorValue; //= UIColor(light: gray500, lightHighContrast: gray700, dark: .white)
+  /** icon that are attached to a label and are only used for decorative purposes */
+  iconSecondary: ColorValue; //= UIColor(light: gray400, lightHighContrast: gray600, dark: gray500, darkHighContrast: gray300, darkElevated: gray400)
+  /** icon color used in disabled state */
+  iconDisabled: ColorValue; //= UIColor(light: gray300, lightHighContrast: gray500, dark: gray600, darkHighContrast: gray400)
+  /** icon appears on top of surfaces that uses `Colors.primary` as background color */
+  iconOnAccent: ColorValue; //= UIColor(light: .white, dark: .black)
+
+  /**
+   * In Dark mode, our system use two sets of background colors -- called base and elevated -- to enhance the perception of depth when one dark interface is layered above another.
+   * The dark base colors are darker, making background interface appear to recede, and the elevate colors are lighter, making foreground interfaces appear to advance
+   */
+
+  surfacePrimary: ColorValue; //= UIColor(light: .white, dark: .black, darkElevated: gray950)
```

Comment:
Reference from the original code, which also had the darkElevated. I thought it would be useful to keep them because you can see where I got the hex values from